_overviews/scala3-migration/tooling-scala2-xsource3.md

```diff
@@ -114,9 +115,17 @@ Example 1:
     + 2
 {% endhighlight %}
 
+Example 2:
+
+{% highlight scala %}
+class C(x: Int) {
+  def x(): Int = x // allowed in Scala 2, double definition error in Scala 3
+}
+{% endhighlight %}
+
 ### Changes affecting binary encoding
 
-As of Scala 2.13.14, there are 3 changes in `-Xsource-features` that affect binary encoding of classfiles:
+As of Scala 2.13.15, there are 3 changes in `-Xsource-features` that affect binary encoding of classfiles:
```

Comment:
is this change intentional?

Reply:
Yes — this PR will be merged at the time we release 2.13.15 (probably this week).

Reply:
But the "Changes affecting binary encoding" section is already online (https://docs.scala-lang.org/scala3/guides/migration/tooling-scala2-xsource3.html#changes-affecting-binary-encoding). Or was that section added too soon and not part of 2.13.14 ?

Reply:
Oh, I see what you mean. 2.13.15 doesn't add any new changes affecting binary encoding. So bumping the version number is only intended to indicate that the information hasn't fallen out date, it is still accurate for 2.13.15.